python3/ltk/apicalls.py

```diff
@@ -405,6 +406,11 @@ def document_update(self, document_id, file_name=None, doc_metadata={}, **kwargs
             log_api('PATCH', uri, r)
         except requests.exceptions.ConnectionError:
             self.handleError()
+        r = Response()
+        r.code = "expired"
+        r.error_type = "expired"
+        r.status_code = 202
+        r._content = b'{ "next_document_id" : "tekashiasdfasdf", "messages" : [ "Failed ya scrub" ]}'
         return r
```

Comment:
I'm assuming we are keeping this for now until we get actual responses? Or comment out if you meant to take it out